duckduckgo/iOS
# move AppHTTPSUpgradeStore to BSK

Task/Issue URL: https://app.asana.com/0/414235014887631/1204208308360926/f
BSK PR: https://github.com/duckduckgo/BrowserServicesKit/pull/267
CC:

**Description**:
- moved AppHTTPSUpgradeStore to BSK
- disabled (new?) swiftlint rule blanket_disable_command

**Steps to test this PR**:
1. Validate BSK is updated from config, HTTPS Upgrade works
2. Validate no data races present (see macOS PR steps: https://github.com/duckduckgo/macos-browser/pull/1042)
